.github/ISSUE_TEMPLATE/design_proposal.md

```diff
@@ -0,0 +1,46 @@
+---
+name: 🤔 Design proposal
+about: Collaborate on a design for a feature/fix or other change
+labels: design-proposal
+---
+
+<!--
+This template is useful to build consensus about whether work should be done, and if so, the high-level shape of how it should be approached. Use this before fixating on a particular implementation.
+-->
+
+## Summary
+
+1-2 sentences. Say what this is about.
+
+## Motivation and goals
+
+1-2 paragraphs, or a bullet-pointed list. What existing pain points does this solve? What evidence shows it's valuable to solve this?
+
+## In scope
+
+A list of major scenarios, perhaps in priority order.
+
+## Out of scope
+
+Scenarios you explicitly want to exclude.
```

Comment:
Same comment as above.... is feature an appropriate descriptor here?

Reply:
Great comments Josh!! Will fix in later iteration!